definitions/variable/technology/electricity-grid.yaml

```
@@ -28,13 +28,27 @@
 - Network|Electricity|Active Power Flow:
     description: Active power flow through a line
     unit: MW
+- Network|Electricity|Active Power Flow Generation Contribution:
```

Suggest to implement a clearer hierarchical structure in the variable name
```suggestion
- Network|Electricity|Active Power Flow|Generation Contribution:
```